build-system/tasks/e2e/puppeteer-controller.js

```diff
@@ -352,7 +352,7 @@ class PuppeteerController {
 
   /**
    * @param {!ElementHandle<!PuppeteerHandle>} handle
-   * @return {!Promise<!{x: number, y: number, height: number. width: number}>}
+   * @return {!Promise<!DomRect>}
```

Comment:
Because DOMRect is still experimental (not supported by Edge or IE) can we keep the original DomRectDef?

https://developer.mozilla.org/en-US/docs/Web/API/DOMRect/DOMRect

Reply:
Added additional properties to `DomRectDef`, and made sure not to use `x`/`y` directly for IE.